### DISCUSSION TITLE
Media Could not be played.

### DISCUSSION THREAD

**Discussion by prajinkhadka** [qutebrowser/qutebrowser#7450] [Q&A] [answered]
I am using qutebrowser on Mac M1 air. 
But, I cannot play the youtube live stream or, embeddd vides in twitter. 

What are other dependencies ?

**Comment by The-Compiler** [comment, marked as answer]
See #7019 - similarly to the situation on Windows, macOS builds come without proprietary codec support. You *might* be able to run from source against a Homebrew-installed Qt there, not sure how the codec solution looks there nowadays.

**Reply by mschilli87** [reply]
Ha! You beat me by a second or so. :laughing:

**Comment by mschilli87** [comment]
How did you install qutebrowser? Please see https://github.com/qutebrowser/qutebrowser/blob/master/doc/install.asciidoc#on-macos and note specifically the following sentence:

> The binary release ships with a QtWebEngine built without proprietary codec support. To get support for e.g. h264/mp4 videos, you’ll need to build QtWebEngine from source yourself with support for that enabled.

As for *how* to install QtWebEngine with support for proprietary codecs on a proprietary operating system, this is IMHO definitely the wrong place to ask for free support.
